DataFormats/EgammaReco/src/EgTrigSumObj.cc

```diff
@@ -0,0 +1,75 @@
+#include "DataFormats/EgammaReco/interface/EgTrigSumObj.h"
+
+#include "DataFormats/RecoCandidate/interface/RecoEcalCandidate.h"
+#include "DataFormats/EgammaReco/interface/SuperCluster.h"
+#include "DataFormats/EgammaReco/interface/ElectronSeed.h"
+#include "DataFormats/GsfTrackReco/interface/GsfTrack.h"
+
+reco::EgTrigSumObj::EgTrigSumObj(float energy, float pt, float eta, float phi)
+    : energy_(energy), pt_(pt), eta_(eta), phi_(phi), hasPixelMatch_(false) {}
+
+reco::EgTrigSumObj::EgTrigSumObj(const reco::RecoEcalCandidate& ecalCand)
+    : energy_(ecalCand.energy()),
+      pt_(ecalCand.pt()),
+      eta_(ecalCand.eta()),
+      phi_(ecalCand.phi()),
+      hasPixelMatch_(false),
+      superCluster_(ecalCand.superCluster()) {}
+
+void reco::EgTrigSumObj::setSeeds(reco::ElectronSeedRefVector&& seeds) {
+  seeds_ = seeds;
```

Comment:
```suggestion
  seeds_ = std::move(seeds);
```
to avoid a copy

Reply:
thanks!

Reply:
I also removed the &&  to take it by value here as well.